.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+  "presets": [
+    ["@babel/preset-env", {
+      "targets": {
+        "browsers": ["last 2 versions"]
```

Comment:
do we want last 2 versions ?  I'd assume you'd want browserslist's `"defaults"`

https://github.com/browserslist/browserslist#best-practices

Reply:
`last 2 versions` only because that's what clay-kiln uses. `defaults` might be better anyways since it's more broad.

Reply:
ah - I didn't realize that was in sync with clay-kiln.  Using whatever clay-kiln has is probably best then to keep things in sync :)

Reply:
or defaults haha - maybe the clay team will have an opinion

Reply:
Anything that supports the version clay-kiln uses is fine. `defaults` sounds like a good choice.